internal/ethapi/api.go

```diff
@@ -63,10 +63,9 @@ func NewPublicEthereumAPI(b Backend) *PublicEthereumAPI {
 	return &PublicEthereumAPI{b}
 }
 
-// GasPrice returns a suggestion for a gas price.
+// GasPrice returns 1 gwei always. Rationale: https://github.com/ethereum-optimism/roadmap/issues/715
```

Comment:
https://github.com/ethereum-optimism/roadmap/issues/715 is quite bare — is that the right link?

Reply:
Supposed to be updated with the decision from https://github.com/ethereum-optimism/roadmap/issues/777